app.py

```diff
@@ -73,7 +73,7 @@ def experience():
 
     return jsonify({})
 
-@app.route('/resume/education', methods=['GET', 'POST'])
+@app.route('/resume/educations', methods=['GET', 'POST'])
```

Comment:
I agree that this is the standard best practice. But, I'd say having consistency in the project is more important than following standard practices. 

Of course, we can update this route name if we do it for all, but I'd say for now let's leave it as it is. You can create a bug issue to track this though.

What do you think?

Reply:
Yes make sense. Will update